common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
> The tracing information has to round trip through Kafka; what's the cost of doing that? Once we have the benchmarking strategy established in the open it makes it easier to then speak about the effect of adding tracing on the critical path in this way.

@rabbah does the sample message shared by @sandeep-paliwal help with the concerns for cost and benchmarking strategies ? Are we worried for introducing these 3 extra fields ?